util/file.go

```diff
@@ -253,20 +256,68 @@ func JoinTerraformModulePath(modulesFolder string, path string) string {
 	return fmt.Sprintf("%s//%s", cleanModulesFolder, cleanPath)
 }
 
-// Returns true if the given path contains the given folder name.
-//
-// Examples:
-//
-// PathContains("/foo/bar", "foo") => returns true
-// PathContains("/foo/bar", "baz") => returns false
-func PathContains(path string, folderName string) bool {
-	pathParts := strings.Split(path, string(filepath.Separator))
+type fileManifest struct {
```

Comment:
Please add comments explaining what this is, why it exists, and how it's meant to be used.

Reply:
done :)